Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -301,6 +319,20 @@ namespace AzToolsFramework
 
     AZStd::vector<AzToolsFramework::ActionOverride> PaintBrushManipulator::PopulateActionsImpl()
     {
+        if (m_pickEntitySelectionMode)
+        {
+            ActionOverride backAction = CreateBackAction(
+                "Stop Entity Pick Mode",
+                "Exit out of the entity pick mode",
+                []()
+                {
+                    AzToolsFramework::EditorPickModeRequestBus::Broadcast(&AzToolsFramework::EditorPickModeRequests::StopEntityPickMode);
+                });
+
+            backAction.SetEntityComponentIdPair(m_ownerEntityComponentId);
+            return { backAction };
+        }
```

Comment:
Cool to see you could handle specifying the new shortcut in this way :+1: (tagging @AMZN-daimini for visibility as this will be one more thing to account for in the Action Manager updates)

Reply:
We were chatting about this yesterday, we may need to introduce a priority system in the HotKey Manager for situations like this